src/index.html

```diff
@@ -1,13 +1,56 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg" 
+        alt="Apple Retina Monoblock" 
+        width="160px" height="134px" 
+        class="card__image"
+      >
```

Comment:
if you write like that, then each attribute should be on a new line
```suggestion
      <img 
        src="./images/imac.jpeg" 
        alt="Apple Retina Monoblock" 
        width="160px" 
        height="134px" 
        class="card__image"
      >
```